src/test/scala/it/pagopa/interop/purposeprocess/RiskAnalysisValidationSpec.scala

```diff
@@ -127,6 +220,34 @@ class RiskAnalysisValidationSpec extends AnyWordSpecLike {
       )
     }
 
+    "succeed only formal (fail if a provided answer depends on an existing field with an unexpected value)" in {
```

Comment:
```suggestion
    "succeed only formal (complete validation should fail because provided answer depends on an existing field with an unexpected value)" in {
```